pkg/cli/install.go

```diff
@@ -29,10 +29,10 @@ acorn install`,
 }
 
 type Install struct {
-	SkipChecks bool `usage:"Bypass installation checks"`
-
-	Image  string `usage:"Override the default image used for the deployment"`
-	Output string `usage:"Output manifests instead of applying them (json, yaml)" short:"o"`
+	SkipChecks bool   `usage:"Bypass installation checks"`
+	Quiet      bool   `usage:"Only output errors encountered during installation"`
```

Comment:
nit: is there a reason not to add `-q` as a shorthand?

```suggestion
	Quiet      bool   `usage:"Only output errors encountered during installation" short:"q"`
```

Reply:
After you called this out, I looked at the rest of the install flags and most don't have shorthands - only three do: help, output, and `-m, --workload-memory-default`.

`help` and `output` make sense because they're kind of meta, but given that the rest don't have shorthands, I don't want to introduce it for this. I'd rather consider doing it holistically in the future. `-m` kind of feels like the odd man out at this point